dev/src/DocFx/Node/XrefTrait.php

```diff
@@ -179,14 +188,19 @@ private function replaceUidWithLink(string $uid, string $name = null): string
             case 0 === strpos($uid, '\Google\Type\\'):
                 $extLinkRoot = 'https://googleapis.github.io/common-protos-php#';
                 break;
+            case 0 === strpos($uid, '\GuzzleHttp\Promise\PromiseInterface'):
+                $extLinkRoot = 'https://docs.aws.amazon.com/aws-sdk-php/v3/api/class-GuzzleHttp.Promise.Promise.html';
+                break;
             default:
                 $extLinkRoot = '';
         }
 
         // Create external link
         if ($extLinkRoot) {
-            $path = str_replace(['::', '\\', '()'], ['#method_', '/'], $name);
-            return sprintf('<a href="%s">%s</a>', $extLinkRoot . $path, $name);
+            if (str_starts_with('\Google', $uid)) {
```

Comment:
This is a bug - these parameters need to be reversed
```suggestion
            if (str_starts_with($uid, '\Google')) {
```